install/python-installer/scripts/traf_user.py

```diff
@@ -103,7 +105,9 @@ def run():
     nodes = dbcfgs['node_list'].split(',')
     trafodion_config = """
 export TRAF_HOME="%s"
-export TRAF_VAR=$TRAF_HOME/tmp
+export TRAF_VAR="%s"
+export TRAF_CONF="%s"
+export TRAF_LOG="%s"
 export MY_SQROOT=$TRAF_HOME # for compatibility
```

Comment:
I think it's time to remove My_SQROOT, please confirm.

Reply:
Yes. Again unrelated to this change, but it makes sense to me that we no longer need that compatibility setting.  That suggestion is localized, so I can include removing it in this change.